graphql/database/invitedUsers/InvitedUsersModel.js

```diff
@@ -0,0 +1,75 @@
+import BaseModel from '../base/BaseModel'
+import types from '../fieldTypes'
+import tableNames from '../tables'
+import { INVITED_USERS } from '../constants'
+import { permissionAuthorizers } from '../../utils/authorization-helpers'
+
+/*
+ * @extends BaseModel
+ */
+class InvitedUsers extends BaseModel {
+  static get name() {
+    return INVITED_USERS
+  }
+
+  static get hashKey() {
+    return 'inviterId'
+  }
+
+  static get rangeKey() {
+    return 'invitedEmail'
+  }
+
+  static get tableName() {
+    return tableNames.invitedUsers
+  }
+
+  static get schema() {
+    const self = this
+    return {
+      inviterId: types
+        .string()
+        .required()
+        .description(
+          `The unique user ID from our authentication service (Firebase)`
+        ),
+      invitedEmail: types
+        .string()
+        .required()
+        .description(`the email address of a newly invited user`),
+      isSquadInvite: types
+        .boolean()
+        .default(self.fieldDefaults.isSquadInvite)
+        .description(
+          `whether the user is invited to a v4 squad or is a normal referral`
+        ),
+      status: types
```

Comment:
Take or leave this feedback: if the status is only two states, it saves on storage to use an `accepted` boolean.